packages/shared/utils/index.ts

```diff
@@ -125,3 +125,7 @@ export function objectEntries<T extends object>(obj: T) {
 export function getLifeCycleTarget(target?: any) {
   return target || getCurrentInstance()
 }
+
+export function toArray<T>(value: T): T extends readonly any[] ? T : [T] {
```

Comment:
Need nullable check for value like https://github.com/antfu/utils/blob/325c0f13b34ab9605dc9c55f6166e3a229cb0f4c/src/array.ts#L9